golem_messages/message/tasks.py

```diff
@@ -234,25 +234,52 @@ def __init__(self, *args, **kwargs):
 
 @library.register(TASK_MSG_BASE + 1)
 class WantToComputeTask(ConcentEnabled, base.Message):
+    """
+    A computation Offer.
+
+    A Provider sends it directly to a Requestor as a response to the Requestor's
+    Demand (Task) in order to get work (SubTask) to do.
```

Comment:
I'd add that the subtask is expressed as a `TaskToCompute`

Reply:
ok